src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2250,7 +2263,7 @@ namespace NServiceBus.Sagas
         public interface Using<T, M> : NServiceBus.Sagas.IFinder
             where T : NServiceBus.IContainSagaData
         {
-            System.Threading.Tasks.Task<T> FindBy(M message, NServiceBus.Extensibility.ReadOnlyContextBag context);
+            System.Threading.Tasks.Task<T> FindBy(M message, NServiceBus.Persistence.TransactionalStorageSession storageSession, NServiceBus.Extensibility.ReadOnlyContextBag context);
```

Comment:
Since we now have both the session and the context should we introduce a explicit `SagaFinderContext` instead?


So instead of

MyFinder
     storageSession.GetRavenSession().Find(...)

we get

MyFinder
     finderContext.GetRavenSession().Find(...)

?